.github/settings.yml

```
@@ -352,7 +359,7 @@ branches:
       enforce_admins: null
       required_linear_history: null
 
-  # l10n branch for zh contents only
+  # l10n branch for zh-ch contents only
```

For your convenience, I point out all existences of `zh-ch`:

```suggestion
  # l10n branch for zh-cn contents only
```